apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/resources/attribution-template.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO mbi_sets (set_type, mbi)
```

Comment:
This is great for our BCDA needs, and would love to stash this away somewhere, but for the sake of simplicity, wondering if we should commit something simpler to the BFD codebase.  What about
```
[
  {{#SQLValues this 10}}"{{this}}"{{#unless @last}},{{/if}}
  {{/SQLValues}}
]
```

Reply:
also, something like the above would imply that we change the name of the file from `attribution-template.sql` to `attribution-template.json`